client/my-sites/people/invite-people/index.jsx

```diff
@@ -270,6 +271,18 @@ const InvitePeople = React.createClass( {
 	},
 
 	render() {
+		if ( this.props.site && ! get( this.props, 'site.capabilities.promote_users' ) ) {
+			return (
+				<Main>
+					<SidebarNavigation />
+					<EmptyContent
+						title={ this.translate( 'Oops, only administrators can invite other\u00a0people' ) }
```

Comment:
This is really not i18n friendly :smile:

@rickybanister I think we should find a more generic solution for long phrases.